dali/operators/numba_function/numba_func.cc

```diff
@@ -48,6 +48,21 @@ If no setup function provided, the output shape and data type will be the same a
 .. note::
     This operator is experimental and its API might change without notice.
 
+.. warning::
+    The ``run_fn`` and ``setup_fn``, when :ref:`conditional mode <conditional_execution>` is enabled,
+    must not be transformed by the AutoGraph. There are two conditions to prevent that:
+
+        1. The ``run_fn`` and ``setup_fn`` must be defined at a global scope
+           (i.e. outside of ``pipeline_def`` scope).
+
+        2. If the ``run_fn`` and ``setup_fn`` is created within pipeline definition,
+           the factory function must be decorated with
+           :meth:`@do_not_convert <nvidia.dali.pipeline.do_not_convert>`. Otherwise it will be
+           recursively converted when the pipeline definition is traced.
```

Comment:
```suggestion
           recursively converted when the pipeline definition is processed.
```
I don't think the word "traced" means anything for the user. For once, I don't think it's actually _traced_ by autograph (what we do with pipeline_def could be described as "tracing", but that's orthogonal).

Reply:
Done

Reply:
It's a term that I borrowed from AutoGraph and we already use it in other places, but processing should also work.